tests-functional/constants.py

```diff
@@ -21,3 +22,8 @@ class Account:
     password="Strong12345",
     passphrase="test test test test test test test test test test nest junk"
 )
+DEFAULT_DISPLAY_NAME = "Mr_Meeseeks"
```

Comment:
just idea: Perhaps a fixture providing random display name would be nice?

Reply:
AFAIK they're never reused. After used once, backends are removed from the list.

Correct me if I'm wrong @antdanchenko

Reply:
But this can be done in a separate PR anyway